specs/protocol/README.md

```diff
@@ -93,42 +106,80 @@ see [here](#signatures) for more details
+| `name`                   | string                                  | N        | Payment Method name. Expected to be rendered on screen.                                                                                             |
+| `description`            | string                                  | N        | Blurb containing helpful information about the payment method. Expected to be rendered on screen. e.g. "segwit addresses only"                      |
+| `group`                  | string                                  | N        | value that can be used to group specific payment methods together e.g. Mobile Money vs. Direct Bank Deposit                                         |
+| `requiredPaymentDetails` | [JSON Schema](https://json-schema.org/) | N        | A JSON Schema containing the fields that need to be collected in the RFQ's selected payment methods in order to use this payment method.            |
+| `fee`                    | [`DecimalString`](#decimalstring)       | N        | Fee charged to use this payment method. absence of this field implies that there is no _additional_ fee associated to the respective payment method |
+| `min`                    | [`DecimalString`](#decimalstring)       | N        | minimum amount required to use this payment method.                                                                                                 |
+| `max`                    | [`DecimalString`](#decimalstring)       | N        | maximum amount allowed when using this payment method.                                                                                              |
+
+> [!IMPORTANT]
+> `kind` should be a unique identifier of an individual payment method
+
+> [!IMPORTANT]
+> the presence of `min` or `max` in a payment method takes precedence over `min` or `max` defined at the `PaymentDetails` level.
+
+> [!IMPORTANT]
+> If `requiredPaymentDetails` is omitted, then any RFQs submitted for the respective offering must also omit `paymentDetails`. 
+
+#### `PayoutMethod`
+| field                     | data type                               | required | description                                                                                                                                         |
+| ------------------------- | --------------------------------------- | -------- | --------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `kind`                    | string                                  | Y        | Type of payment method (i.e. `DEBIT_CARD`, `BITCOIN_ADDRESS`, `SQUARE_PAY`). considered to be a unique identifier                                   |
+| `estimatedSettlementTime` | uint                                    | Y        | estimated time taken to settle an order. expressed in seconds                                                                                       |
```

Comment:
Super nit: why not millis instead of seconds? I agree that the vast majority of payments with TBDex will be slower than 1 second, but I wonder if a small few may find it useful have very tight settlement times less than one second

Reply:
fair point @diehuxx . went with seconds because anything less than 1 second i considered to be `0` aka "instant" bc i assume no one will be using tbdex for high frequency day trading